automation/src/main/java/org/wso2/carbon/testgrid/automation/file/TestNGTestReader.java

```diff
@@ -18,19 +18,57 @@
 
 package org.wso2.carbon.testgrid.automation.file;
```

Comment:
What is the reason to use 'org.wso2.carbon.testgrid.automation.**file**' package name?

Reply:
This is because this package contains classes for reading files.